easybuild/easyconfigs/m/Meson/Meson-0.49.0-GCCcore-7.3.0-Python-3.6.6.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'Meson'
+version = '0.49.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://mesonbuild.com'
+description = "Meson is a cross-platform build system designed to be both as fast and as user friendly as possible."
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['a0cd55372208730eca5cd7cdff7948e439a006d8587c50c4445be1a0e88b2521']
+
+builddependencies = [
+    ('binutils', '2.30'),
+]
+
+dependencies = [
+    ('Python', '3.6.6', '-bare'),
```

Comment:
@jose-d Using a `-bare` `Python` as a runtime dependency is not a good idea, since it will make this module incompatible with a non-bare `Python` installation...

What you basically need to pull down `Meson` to `GCCcore` is a stripped-down (non-bare) `Python` at `GCCcore`, see also the recent discussion on the EasyBuild mailing list: https://lists.ugent.be/wws/arc/easybuild/2018-12/msg00029.html

Reply:
@boegel  I see.. I realized this with when digging around `HarfBuzz` - which actually needs `six` as build dependency, so `-bare` would bring lot of fun.

I think ATM I should slow down my effort to push the `GTK+3` into upstream - I'll keep it in our  _site-specific_ repo, and wait untill there will be some common agreement what to do with Python(-core|-GCCcore) thing.

I'll close all gtk3-related non-GCCcore PRs for now, and let's reopen this topic after the `EBUM`.